README.md

```diff
@@ -10,7 +10,7 @@ A [Dart](https://www.dartlang.org) plugin for [Atom](https://atom.io).
 
 - install [Atom](https://atom.io/)
 - install the [linter][] package
-- install this [dartlang][] package
+- install this [dartlang][] package with `apm install dartlang`
```

Comment:
I think most users would be better off installing through the UI rather than the command line. This guy:

<img width="516" alt="screen shot 2015-08-06 at 11 50 53 am" src="https://cloud.githubusercontent.com/assets/1269969/9120497/6e9b89c2-3c31-11e5-8fed-76caf67df4fd.png">

That said, we should have better getting-started docs for this plugin, including installation instructions and some tips for accessing useful dart and atom features.